### DISCUSSION TITLE
Spurious contributors on repository front page

### DISCUSSION THREAD

**Discussion by loxodes** [community/community#23227] [New to GitHub]
<p>The front page of my <a href="https://github.com/loxodes/vna" rel="noopener nofollow ugc">loxodes/vna</a> repo lists a spurious contributor which is not present on the contributor insights tab.</p>
<p>This is may be result of me committing from a computer without setting a name and email. I scrubbed the git commit author history a few months ago but the spurious contributor remains.</p>
<p>On this thread, Github staff clearing cached data on a repository resolved a similar issue:<br>
<a href="https://github.community/t/bug-new-contributors-section-does-sometimes-not-match-the-contributors-tab-under-insights/121576">[BUG] New “Contributors” section does sometimes not match the “Contributors” tab under “Insights”</a></p>
<p>Please clear cached contributor data on my loxodes/vna repo</p>
<p>Thanks</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/loxodes">@loxodes</a>, welcome to the GitHub Support Community! I’ve cleared that cache for you so that extra contributor should be gone from your repo homepage now :+1:</p>
